content/questions/object-clone-stringify/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Object Cloning (JSON.parse + JSON.stringify)
+tags:
+  - object
+  - cloning
+order: 28
+date: '2019-10-05'
+answers:
+  - 'true true true true'
+  - 'true true true false'
+  - 'true true false true' // correct
+  - 'false false false false'
+
+---
+
+Consider objects `a` and `b` below. What gets logged?
+
+```javascript
+const a = { 
+  stringField: 'Joe',
+  numberField: 123,
+  dateField: new Date('1995-12-17T03:24:00')
+  nestedField: {field: 'Nested'}
+ };
+const b = JSON.parse(JSON.stringify(a));
+console.log(a.stringField === b.stringField,
```

Comment:
Needs trailing commas for all these logged items. Also, can you indent the new lines?